lib/tasks/stats.rake

```diff
@@ -102,80 +109,37 @@ module Reporting
   def started_learning
-    User.registered.map { |u| u.module_time_to_completion.keys }.count(&:present?)
+    User.all.map { |u| u.module_time_to_completion.keys }.count(&:present?)
   end
 
   # Number of users not started
   def not_started(mod)
-    User.registered.map { |u| u.module_time_to_completion[mod.name] }.count(&:nil?)
+    User.all.map { |u| u.module_time_to_completion[mod.name] }.count(&:nil?)
   end
 
   # Number of users in progress
   def in_progress(mod)
-    User.registered.map { |u| u.module_time_to_completion[mod.name] }.compact.count(&:zero?)
+    User.all.map { |u| u.module_time_to_completion[mod.name] }.compact.count(&:zero?)
   end
 
   # Number of users completed
   def completed(mod)
-    User.registered.map { |u| u.module_time_to_completion[mod.name] }.compact.count(&:positive?)
+    User.all.map { |u| u.module_time_to_completion[mod.name] }.compact.count(&:positive?)
   end
 
-  # Number of users (total)
-  def engagement(mod)
+  # Number of users started
+  def started(mod)
     in_progress(mod) + completed(mod)
   end
-
-  # @see ContentPagesController#track_events
-  # ----------------------------------------------------------------------------
-
-  # Number of 'module_start' events
-  def module_start_event(mod)
-    Ahoy::Event.where(name: 'module_start').where_properties(training_module_id: mod.name).count
-  end
-
-  # Number of 'module_complete' events
-  def module_complete_event(mod)
-    Ahoy::Event.where(name: 'module_complete').where_properties(training_module_id: mod.name).count
-  end
-
-  # Number of 'confidence_check_complete' events
-  def confidence_check_complete_event(mod)
-    Ahoy::Event.where(name: 'confidence_check_complete').where_properties(training_module_id: mod.name).count
-  end
-
-  # @see QuestionnairesController#track_events
-  # ----------------------------------------------------------------------------
-
-  # Number of 'confidence_check_start' events
-  def confidence_check_start_event(mod)
-    Ahoy::Event.where(name: 'confidence_check_start').where_properties(training_module_id: mod.name).count
-  end
-
-  # Number of 'summative_assessment_start' events
-  def summative_assessment_start_event(mod)
-    Ahoy::Event.where(name: 'summative_assessment_start').where_properties(training_module_id: mod.name).count
-  end
-
-  # @see AssessmentResultsController#track_events
-  # ----------------------------------------------------------------------------
-
-  # Number of PASS 'summative_assessment_complete' events
-  def pass_summative_assessment_complete_event(mod)
-    Ahoy::Event.where(name: 'summative_assessment_complete').where_properties(training_module_id: mod.name, success: true).count
-  end
-
-  # Number of FAIL 'summative_assessment_complete' events
-  def fail_summative_assessment_complete_event(mod)
-    Ahoy::Event.where(name: 'summative_assessment_complete').where_properties(training_module_id: mod.name, success: false).count
-  end
 end
+# :nocov:
```

Comment:
This rake task is here to give a quick sanity check to data we expect to see in our dashboard. Until the dashboard is up and running, being fed automatically, this task gave us a quick and easy way to tally some stats to share across the team. As it is not component for the app or expected to remain once the dashboard is working it is just here for speed and convenience. As a result, I've wrapped it with tags to prevent it from impacting the coverage score and also make it clear that no test coverage is present or necessary.